### DISCUSSION TITLE
未来Umi是否会弃用Dva？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [umijs/umi#10451] [Q&A]
未来Umi是否会弃用Dva插件？因为已经不再维护。

我们知道Umi max 默认安装了Dva插件和valtio插件。

1.Dva主要是否解决状态便捷使用的问题？
2.这里有valtio插件之后，是否不使用Dva插件也能很方便地开发项目？

**Comment by SilentFlute** [comment]
我觉得未来umi应该是会弃用dva, 作者建议使用[valtio](https://umijs.org/docs/max/valtio)或者[jotai](https://github.com/pmndrs/jotai): https://github.com/umijs/umi/issues/10680#issuecomment-1456150552
